impl/pkg/service/pkarr.go

```diff
@@ -27,27 +31,46 @@ func NewPKARRService(cfg *config.Config, db *storage.Storage) (*PKARRService, er
 	if db == nil && !db.IsOpen() {
 		return nil, util.LoggingNewError("storage is required be non-nil and to be open")
 	}
-	dht, err := dht.NewDHT(cfg.DHTConfig.BootstrapPeers)
+	d, err := dht.NewDHT(cfg.DHTConfig.BootstrapPeers)
 	if err != nil {
 		return nil, util.LoggingErrorMsg(err, "failed to instantiate dht")
 	}
-	return &PKARRService{
-		cfg: cfg,
-		db:  db,
-		dht: dht,
-	}, nil
+	scheduler := dhtint.NewScheduler()
+	service := PKARRService{
+		cfg:       cfg,
+		db:        db,
+		dht:       d,
+		scheduler: &scheduler,
+	}
+	if err = scheduler.Schedule(cfg.PKARRConfig.RepublishCRON, service.republish); err != nil {
```

Comment:
The way this is setup, there will be a republish job for each instance that's running did-dht. Is that intentional?

Reply:
intended to run as a single instance